app.js

```diff
@@ -17,13 +17,13 @@ app.set('view engine', 'jade');
 
 // uncomment after placing your favicon in /public
 
-var corsOptions = {
-    origin: '*.cyclomedia.com',
-    credentials: true,
-    optionsSuccessStatus: 200 // some legacy browsers (IE11, various SmartTVs) choke on 204
-}
+// var corsOptions = {
```

Comment:
Can you just delete these? I'm trying to get us into the habit of not leaving in commented code. If we need to bring stuff back, that's what git's for.